python-avd/pyavd/_eos_designs/structured_config/metadata/__init__.py

```diff
@@ -31,19 +32,24 @@ def metadata(self) -> None:
             dc_name=self.inputs.dc_name,
             fabric_name=self.shared_utils.fabric_name,
             serial_number=self.shared_utils.serial_number,
-            validate_no_errors_period=self.inputs.logging_settings.validate_no_errors_period,
         )
-        exclude_as_extra_fabric_validation_target = default(
-            self.shared_utils.node_config.exclude_as_extra_fabric_validation_target,
-            self.shared_utils.node_type_key_data.exclude_as_extra_fabric_validation_target,
-        )
-        if exclude_as_extra_fabric_validation_target:
-            self.structured_config.metadata.exclude_as_extra_fabric_validation_target = exclude_as_extra_fabric_validation_target
         self._set_cv_tags()
         self._set_cv_pathfinder()
         if self.shared_utils.digital_twin:
             self._set_digital_twin()
-        self.structured_config.metadata.validate_hardware = self.shared_utils.platform_settings.validate_hardware
+
+        # Logic for validate hardware
+        if not self.shared_utils.platform_settings.feature_support.hardware_validation:
+            self.structured_config.metadata.validate_hardware.enabled = False
+        if self.shared_utils.node_config.validation_profile:
```

Comment:
Return if there is no validation profile - then continue
```suggestion
        if not self.shared_utils.node_config.validation_profile:
            return
```